src/main.ts

```diff
@@ -24,7 +24,33 @@ export default class AdvancedCloseTab extends Plugin {
         this.detachLeafIfUnpinned(leaf);
       },
     });
+    
+    this.addCommand({
+      id: 'advanced-close-tab-close-current-tab-if-not-in-sidebar',
```

Comment:
It seems that it is not necessary to include `advanced-close-tab-` here, as Obsidian automatically prefixes the Plugin ID (I didn't know this when I created this plugin, so the other commands are not like that...).

https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Don't+include+the+plugin+ID+in+the+command+ID

So, could you please change it to `close-current-tab-if-not-in-sidebar`?